server/schemas/internalapi/v1/schema.proto

```
@@ -0,0 +1,91 @@
+syntax = "proto3";
+
+package reearth.visualizer.v1;
```

Buf (1.47.2)

Files with package "reearth.visualizer.v1" must be within a directory "reearth/visualizer/v1" relative to root but were in directory "server/schemas/internalapi/v1".

(PACKAGE_DIRECTORY_MATCH)